kibana.md

```diff
@@ -107,18 +107,69 @@ Notice how we need to type in the format `file://` + absolute path (yes, with th
 
 If you are using Kibana 7.9.x or newer, you need **an extra post-installation step**. This will slightly modify some core Kibana files.
 
+#### Before Kibana 8.15.0
+
+```bash
+# Patch Kibana core files 
+node/bin/node plugins/readonlyrestkbn/ror-tools.js patch
+```
+
+#### For Kibana 8.15.0 and never
+
+##### For Linux 
+
 ```bash
 # Patch Kibana core files 
-$ node/bin/node plugins/readonlyrestkbn/ror-tools.js patch
+node/glibc-217/bin/node plugins/readonlyrestkbn/ror-tools.js patch
+```
+
+##### For macOS
+
+```bash
+# Patch Kibana core files 
+node/default/bin/node plugins/readonlyrestkbn/ror-tools.js patch
```

Comment:
I'm not sure why the path is different on MacOS. I thought that regardless of the installation method, the internal structure of folders is the same (excluding the conf folder).